components/scripts/lib/gradle.sh

```diff
@@ -64,6 +64,11 @@ invoke_gradle() {
     die "ERROR: The experiment cannot continue because of a non-recoverable failure while running the build: $(cat "${EXP_DIR}/errors.txt")"
   fi
 
```

Comment:
Did we forget to also add this for Maven? (not strictly related to this PR)

Reply:
Maven does not write to or create an `errors.txt` file so there is nothing to check for.

Reply:
Got it.